lib/diagnostic.rb

```diff
@@ -7,36 +7,36 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+    ar.max_by(&:length)
```

Comment:
This is correct but you should have `list.max_by(&:length)` because you are passing in `list` as the parameter, not `ar`.